pytorch-job/pytorch-operator/overlays/application/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+- ../../base
+resources:
+- application.yaml
+commonLabels:
```

Comment:
will spec.selector of `Application` be filled with these commonLabels? I know deployment spec.selector can be filled automatically, but now sure `Application`

Reply:
yes Application will have these labels that match the kustomize target the application.yaml is located in